docs/server/source/server-reference/bigchaindb-cli.md

```diff
@@ -80,22 +80,27 @@ configuration file as documented under
 **This is an experimental feature. Users are advised not to use it in production.**
 
 
-Add, update, or remove a validator from the validators set of the local node. The command implements [3/UPSERT-VALIDATORS](https://github.com/bigchaindb/BEPs/tree/master/3), check it out if you need more details on how this is orchestrated.
+Manage elections to add, update, or remove a validator from the validators set of the local node. The commands implement [21/UPSERT-VALIDATORS](https://github.com/bigchaindb/BEPs/tree/master/21), check it out if you need more details on how this is orchestrated.
```

Comment:
I'd change "The commands implement [21/UPSERT-VALIDATORS]..." to "The upsert-validator subcommands implement [BEP-21]..."

(We now refer to BEPs with shortnames like BEP-21. See https://github.com/bigchaindb/BEPs/pull/51 )

Reply:
Done